docker-compose.lightweight.yml

```diff
@@ -0,0 +1,69 @@
+version: '3'
+
+volumes:
+  streaming:
+    driver: local
+  database:
+    driver: local
+  fcrepo:
+    driver: local
+  work:
+    driver: local
+  solr:
+    driver: local
+  redis:
+    driver: local
+
+services:
+  db:
+    image: postgres:9.6-alpine
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_PASSWORD: mysecretpassword
+      POSTGRES_USER: avalon
+    volumes:
+      - database:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+
+  fcrepo:
+    image: ualbertalib/docker-fcrepo4
+    volumes:
+      - fcrepo:/fcrepo4-data
+    ports:
+      - "8984:8080"
+
+  solr:
+    image: solr:6.6
+    ports:
+      - "8983:8983"
+    volumes:
+      - solr:/opt/solr/server/solr/mycores
+      - ./solr/config:/config
+      - ./solr/scripts/solr-precreate-avalon.sh:/docker-entrypoint-initdb.d/solr-precreate-avalon.sh
+
+  matterhorn:
+    image: avalonmediasystem/matterhorn
+    volumes:
+      - ./masterfiles:/masterfiles
+      - streaming:/streamfiles
+      - work:/work
+    ports:
+      - "8080:8080"
+  hls:
+    image: avalonmediasystem/nginx
+    network_mode: "host"
```

Comment:
The network mode is set to `host` here so that the `hls` server can contact the main app server (running on localhost) to determine authorization to view content.